place_tracker/lib/generated_plugin_registrant.dart

```diff
@@ -0,0 +1,16 @@
+//
+// Generated file. Do not edit.
+//
```

Comment:
No need to commit this file, you can safely add it to some `.gitignore`

Reply:
Makes sense, removed